### PR TITLE
[SYCLomatic] Make PTX code works anyway on CUDA Backend

### PR DESCRIPTION
**Pull request by yihwang-nv**

### CODE REVIEW

File: clang/lib/Parse/ParseStmtAsm.cpp [hunk 1]
```diff
@@ -861,6 +861,8 @@ StmtResult Parser::ParseAsmStatement(bool &msAsm) {
     SkipUntil(tok::r_paren, StopAtSemi);
     return StmtError();
   }
+  if (StringRef(Tok.getLocation().printToString(Actions.getSourceManager())).contains("asm.cu"))
```
Comment (wenhuiNi) [inline clang/lib/Parse/ParseStmtAsm.cpp]: remove debug code

Comment (yihwang-nv) [reply clang/lib/Parse/ParseStmtAsm.cpp]: removed

**Review by AndyCHHuang — APPROVED** [review]
LGTM

**Review by zhimingwang36 — APPROVED** [review]
LGTM

**Pull request merged**